uqcs/app.py

```diff
@@ -77,6 +76,8 @@ def user_from_request(form):
             info["undergrad"] = form['degreeType'].lower() == 'undergrad'
         if 'degree' in form:
             info['program'] = form['degree'][:100]
+        if 'majors' in form:
+            info['majors'] = '|'.join(form.getlist('majors'))[:1000]
```

Comment:
sqlalchemy is magic and postgres arrays are actually pretty chill if you wanna do that instead - you can literally just pass the list to the constructor

https://docs.sqlalchemy.org/en/13/dialects/postgresql.html#sqlalchemy.dialects.postgresql.ARRAY

Reply:
It is indeed ✨magic ✨